shell/platform/windows/platform_handler.h

```diff
@@ -33,6 +40,14 @@ class PlatformHandler {
 
   virtual ~PlatformHandler();
 
+  // String values used for encoding/decoding exit requests.
+  static constexpr char kExitTypeCancelable[] = "cancelable";
+  static constexpr char kExitTypeRequired[] = "required";
+
+  // Send a request to the framework to test if a cancelable exit request
+  // should be canceled or honored.
+  virtual void RequestAppExit(AppExitType exit_type, UINT exit_code, HWND hwnd);
```

Comment:
Consider updating the comment to mention that `HWND` is the window that initiated the app exit.